CHANGELOG.md

```diff
@@ -137,6 +139,7 @@ Compactor now properly handles partial block uploads for all operation like rete
 
 - [#1852](https://github.com/thanos-io/thanos/pull/1852) Add support for `AWS_CONTAINER_CREDENTIALS_FULL_URI` by upgrading to minio-go v6.0.44
 - [#1854](https://github.com/thanos-io/thanos/pull/1854) Update Rule UI to support alerts count displaying and filtering.
+<<<<<<< HEAD
```

Comment:
Seems like there's a remnant here of a merge conflict being solved. Could you remove this?

Reply:
Ooopsie !